lesson2/2_8.js

```diff
@@ -0,0 +1,38 @@
+// Создать объект human со следующими возможностями:
+// При записи значения в свойство fullName имя и фамилия должны записываться в отдельные свойства firstName и lastName
+// При чтении значения fullName оно должно получаться из объединения firstName и lastName
+// При задании значения свойства dateOfBirth должно так же устанавливаться свойство age в зависимости от разницы года рождения и текущего года
+
+const human = Object.create ({} , {
+  fullName: {
+    set: function (valye) { 
+      [this.firstName, this.lastName] = valye.split(' ')
+    },
+    get () { return [this.firstName, this.lastName].filter(Boolean).join(' ')
+      }
+    },
+
+dateOfBirth: {
+   set: function (valye) {
+     [this.dayOfbirth, this.monthOfbirth, this.yearOfbirth] =  valye.split('.')
+   },
+     get () { return [this.dayOfbirth, this.monthOfbirth, this.yearOfbirth].join('.')
+            }
+    },
+age: { 
+   get() { age = Date().split(' ')[3] - this.yearOfbirth
+   return 'age ' + age 
+      }
+    }  
```

Comment:
Все правильно, однако замечу что в случае со свойством `age` можно не объявлять для него геттер, а просто хранить как значение, которое будет установлено при вызове сеттера даты:

```suggestion
dateOfBirth: {
   set: function (valye) {
     [this.dayOfbirth, this.monthOfbirth, this.yearOfbirth] =  valye.split('.');
     this.age = new Date().getFullYear() - parseInt(this.yearOfbirth)
   },
     get () { return [this.dayOfbirth, this.monthOfbirth, this.yearOfbirth].join('.')
            }
    }
```